pikaday.js

```diff
@@ -1082,13 +1104,23 @@
             }
 
             for (var c = 0; c < opts.numberOfMonths; c++) {
-                randId = 'pika-title-' +
-                  Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 2);
-
-                html += '<div class="pika-lendar">' + renderTitle(this, c, this.calendars[c].year, this.calendars[c].month, this.calendars[0].year, randId) + this.render(this.calendars[c].year, this.calendars[c].month, randId) + '</div>';
+                randId = 'pika-title-' + c;
+                var pikaDaysContainer = this.render(this.calendars[c].year, this.calendars[c].month, randId);
+                var pikaTitleContainer = renderTitle(this, c, this.calendars[c].year, this.calendars[c].month, this.calendars[0].year, randId);
+
+                // update the days container only
+                var dayContainerSelector = this.el.querySelector('div#pika-days-' + c);
+                if (dayContainerSelector) {
+                    dayContainerSelector.innerHTML = pikaDaysContainer;
+                } else {
+                    html += '<div class="pika-lendar">' + pikaTitleContainer + '<div id="pika-days-' + c + '">' + pikaDaysContainer + '</div></div>';
```

Comment:
Wrap the `pikaDaysContainer` inside the first time drawing, so far when we can update its content with `querySelector`.